archinstall/default_profiles/desktops/river.py

```diff
@@ -0,0 +1,23 @@
+from typing import override
+
+from archinstall.default_profiles.profile import GreeterType, ProfileType
+from archinstall.default_profiles.xorg import XorgProfile
+
+
+class riverProfile(XorgProfile):
+	def __init__(self) -> None:
+		super().__init__('River', ProfileType.WindowMgr, description='')
+
+	@property
+	@override
+	def packages(self) -> list[str]:
+		return [
+			'foot',
```

Comment:
According to the wiki only `river` is mentioned that needs to be installed https://wiki.archlinux.org/title/River, what are these other packages for?

Reply:
in the example init it recommends foot https://codeberg.org/river/river/src/branch/master/example/init#L14